content/visual-testing-handbook/react/es/vtdd.md

```diff
@@ -0,0 +1,335 @@
+---
+title: 'TDD Visual'
+description: 'Escribe tus primeras pruebas visuales'
+---
+
+Ahora que los conceptos básicos fueron cubiertos, veamos los detalles. Este ejemplo muestra la construcción de un estado de un componente `CommentList` usando **TDD Visual** (desarrollo guiado por pruebas visuales) con Storybook.
+
+1. Cree casos de prueba visuales
+2. Consulte las pruebas en Storybook
+3. Desarrolle la implementación
+4. Compare la implementación con el diseño.
+5. Iterar
+
+### Lo que estamos construyendo
+
+`CommentList` es parte de una herramienta de chat para luchadores por la libertad galáctica. Nuestro diseñador nos ha entregado un diseño para las diversas formas en que debe verse la lista de comentarios según los datos y el estado de la aplicación. Nuestro trabajo es garantizar que la lista se muestre correctamente en términos del texto exacto, las imágenes mostradas y el tratamiento visual.
+
+![Especificaciones de diseño de Commentlist](/visual-testing-handbook/visual-testing-handbook-commentlist-design-optimized.png)
+
+### 1. Cree casos de prueba visuales
+
+Inicie el desarrollo guiado por pruebas visuales mediante la creación de casos de prueba. Crearemos tres casos que coincidan con las tres imágenes de arriba. Un desarrollo guiado por pruebas estricto diría que necesitamos desarrollar e implementar un caso de prueba a la vez; depende de usted si cree que esto ayuda a su proceso.
+
+Configuremos el proyecto de ejemplo usando [degit](https://github.com/Rich-Harris/degit) para descargar las plantillas repetitivas necesarias (aplicaciones parcialmente construidas con alguna configuración predeterminada). Ejecute los siguientes comandos:
+
+```shell
+# Clone la plantilla para este tutorial
+npx degit chromaui/visual-testing-handbook-react-template commentlist
+
+cd commentlist
+
+# Instale dependencias
+yarn
```

Comment:
@delfilb if you don't mind leaving the comments in the original form we'd appreciate it. We're aware that it brings the translation closer to home, but until we come up with a good solution for this we'd appreciate if you left them in English.